pvm_2_videos.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+import os
+from pythonosc import dispatcher, osc_server
+import argparse
+from omxplayer.player import OMXPlayer
+import logging
+from logging.handlers import TimedRotatingFileHandler
+import sys
+
+def _init_logger():
+	logger = logging.getLogger("PVM")
+	logger.setLevel(logging.INFO)
+	handler = logging.StreamHandler(sys.stderr)
+	fileHandler = TimedRotatingFileHandler('./log/{:%Y-%m-%d %H:%M:%S}.log'.format(datetime.now()),  when='midnight')
+	handler.setLevel(logging.INFO)
+	formatter = logging.Formatter("%(asctime)s;%(levelname)s;%(message)s",
+                              "%Y-%m-%d %H:%M:%S")
+	fileHandler.setFormatter(formatter)
+	fileHandler.suffix = '%Y_%m_%d.log'
+	logger.addHandler(fileHandler)
+	handler.setFormatter(formatter)
+	logger.addHandler(handler)
+
+_init_logger()
+_logger = logging.getLogger("PVM")
+_logger.info("Logging system initilized in %s", os.getcwd())
+
+# Place your videos in this folder for autostart
+PEFIX_PATH = "/home/pi/Videos/"
+VIDEO_PATH_ONE = "jellyfish720p.mp4"
+VIDEO_PATH_TWO = "jellyfish720p.mp4"
+media1 = ""
+media2 = ""
+IS_FILE_SET = False
+
+def parse_commands(*args):
+	global media1
+	global media2
+	global VIDEO_PATH_ONE
+	global VIDEO_PATH_TWO
+	global IS_FILE_SET
+	command = args[1]
+	_logger.info("Received command: %s", command)
+	if len(args)>2:
+		value = args[2]
+		_logger.info("Received command: %s", str(value))
+		pass
+	# TODO: Create another python file to control two display
+	if command=="file":
+		if "|" not in value:
+			_logger.info("Please pass two video name like 1.mp4,2.mp4.")
+			return
+		VIDEO_PATH_ONE, VIDEO_PATH_TWO = value.split("|")
```

Comment:
the example would be `1.mp4|2.mp4` so that it splits the string into 2, is that it?
I confess I don't like this strategy very much - thinking from the point of view of the future control interface.
Could you do it in a way that the user can pass 1 or 2 arguments, and if it is 2 arguments then it must be for 2 video players? Yes that will mean changing line 45.

Reply:
Could you explain more about this use case?
User input: 1.mp4
Log output: You need to input 2 filenames.
Or
User input: 1.mp4
Log output: We consider 1.mp4|1.mp4